go.mod

```diff
@@ -8,40 +8,42 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/gorilla/mux v1.8.0
 	github.com/jackc/pgconn v1.10.0
+	github.com/jackc/pgproto3/v2 v2.1.1
 	github.com/jackc/pgx/v4 v4.13.0
 	github.com/prometheus/client_golang v1.11.0
+	github.com/stolostron/multicloud-operators-foundation v1.0.0-2021-10-26-20-16-14.0.20220307042847-0001f46a5115
 	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
Why downgrade?  There's also a replace at the bottom to change it back.

Reply:
Pulling `github.com/stolostron/multicloud-operators-foundation` causes that, I think.

Reply:
Should we use the latest version of `multicloud-operators-foundation` to avoid problems with CVEs in the future?  https://github.com/stolostron/multicloud-operators-foundation/tags

Reply:
Thanks for trying!